epochStart/bootstrap/storageProcess_test.go

```diff
@@ -64,37 +56,36 @@ func TestNewStorageEpochStartBootstrap_InvalidArgumentsShouldErr(t *testing.T) {
 		sesb, err := NewStorageEpochStartBootstrap(args)
 		assert.True(t, check.IfNil(sesb))
 		assert.True(t, errors.Is(err, epochStart.ErrNilHasher))
-
 	})
-	t.Run("nil ChanGracefullyClose should err", func(t *testing.T) {
+	t.Run("nil RunTypeComponents should err", func(t *testing.T) {
```

Comment:
Shouldn't we check nil pointers for RequestCreator and NodesCoordCreator? Or are they tested somwhere else?

Reply:
all the parameters are checked in `CreateEpochStartBootstrapper` function, and are covered by tests in `process_test.go`